scripts/zones/Castle_Oztroja/IDs.lua

```diff
@@ -32,6 +32,7 @@ zones[xi.zone.CASTLE_OZTROJA] =
         MEMBERS_LEVELS_ARE_RESTRICTED      = 7208, -- Your party is unable to participate because certain members' levels are restricted.
         FISHING_MESSAGE_OFFSET             = 7268, -- You can't fish here.
         CHEST_UNLOCKED                     = 7439, -- You unlock the chest!
+        TEBHI_ACCEPTS                      = 7450, -- Tebhi happily accepts the
```

Comment:
This will break our update tooling and require manual intervention. We match on the comment, so the comment has to be exactly in line with what our update tooling spits out.

These client extracts are old (ignore the numbers), but they show you what the text _should_ be:

https://github.com/LandSandBoat/UpdateExtractor/blob/main/out/scripts/zones/Castle_Oztroja/Text.lua.raw.txt#L7362

We use `<>` for wildcard while doing these checks, so the comment should be: `Tebhi happily accepts the <item> and leaves.`

Reply:
Didn't know about this!! Thanks 🙏🏼